frontend/TestRun/ResultsTab.svelte

```diff
@@ -0,0 +1,113 @@
+<script>
+    import {onMount} from "svelte";
+
+    let fetching = true;
+    export let id = "";
+    export let test_id = "";
+    let results = [];
+
+
+    const fetchResults = async function () {
+        fetching = true;
+        try {
+            let apiResponse = await fetch(`/api/v1/run/${test_id}/${id}/fetch_results`);
+            let apiJson = await apiResponse.json();
+            if (apiJson.status === "ok") {
+
+                fetching = false;
+                results = apiJson.tables;
+                console.log(results);
+            }
+        } catch (error) {
+            console.log(error);
+        }
+    };
+
+    const durationToStr = (seconds) => {
+        const hours = Math.floor(seconds / 3600);
+        const minutes = Math.floor((seconds % 3600) / 60);
+        const secs = seconds % 60;
+        return `${hours.toString().padStart(2, "0")}:${minutes.toString().padStart(2, "0")}:${secs.toString().padStart(2, '0')}`;
+    };
+
+    const formatValue = (value, type) => {
+        switch (type) {
+        case "FLOAT":
+            return value.toFixed(2);
+        case "INTEGER":
+            return value.toLocaleString();
+        case "DURATION":
+            return durationToStr(value);
+        default:
+            return value;
+        }
+    };
+
+    const getColumnType = (result, columnName) => {
+        return result.meta.columns_meta.find((col) => col.name === columnName).type;
+    };
+
+    onMount(() => {
+        fetchResults();
+    });
+
+    const styleMap = {
+        "PASS": "table-success",
+        "ERROR": "table-danger",
+        "WARNING": "table-warning"
+    };
+
+</script>
+
+<style>
+    th, td {
+        text-align: center;
+    }
+
+    table {
+        width: auto;
+        table-layout: auto;
+    }
+</style>
+
+{#if !fetching}
+    <div class="m-2">
+        {#each results as result}
+            <div class="">
+                <h3>{result.meta.name}</h3>
+                <span>{result.meta.description}</span>
+                <table class="table table-bordered">
+                    <thead class="thead-dark">
+                    <tr>
+                        <th></th>
+                        {#each result.meta.columns_meta as col}
+                            <th>{col.name} [{col.unit}]</th>
```

Comment:
can try something like this instead here
```html
<span class="rounded-pill d-inline-block bg-secondary" style="font-weight: 0.8em">{col.unit}</span>
```

Reply:
I didn't like it. But emphasized a bit - see next screenshot.